3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_irs_mop_up.psql

```diff
@@ -227,23 +260,35 @@ SELECT * FROM
             business_status IN ('Not Eligible', 'No Tasks') AND
             (plans.identifier = zambia_irs_structures.plan_id OR zambia_irs_structures.plan_id IS NULL)
     ) AS inactive_irs_structures ON true
+    LEFT JOIN LATERAL (
+        SELECT 
+			events.id AS latest_event_id,
+			events.event_date AS latest_event_date
+		FROM events
+		WHERE
+			events.event_type = 'irs_sa_decision'
```

Comment:
Here's the one place irs_sa_decision events come in - and we only use their date.